ResNet_constructor.ipynb

```diff
@@ -0,0 +1,1288 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #12.        accuracy_val = train_tools.approx_accuracy(testset, model, device, denominator=4)
</pre><p>Update notebooks so they use latest version the code (e.g., <code>dataset_fraction</code>  instead of <code>denominator</code>  )</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bestetc/May/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='37'/>